src/shaders/hillshade.fragment.glsl

```diff
@@ -0,0 +1,51 @@
+
+#ifdef GL_ES
+precision highp float;
+#else
+#define lowp
+#define mediump
+#define highp
+#endif
```

Comment:
We're defining this in `_prelude.{fragment,vertex}.glsl`, which is automatically prepended to all shaders in both GLJS and GL Native, so there's no need to include this definition here.

Reply:
I have to keep highp precision for hillshade_prepare.fragment for it to work on some android devices.